<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QWidget" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../res.qrc">
    <normaloff>:/res/img/16x16/appicon.png</normaloff>:/res/img/16x16/appicon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QFrame" name="frameButtons">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="buttonSave">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string> Сохранить</string>
        </property>
        <property name="icon">
         <iconset resource="../../res.qrc">
          <normaloff>:/res/img/16x16/settings-save.png</normaloff>:/res/img/16x16/settings-save.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonReload">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string> Сброс изменений</string>
        </property>
        <property name="icon">
         <iconset resource="../../res.qrc">
          <normaloff>:/res/img/16x16/settings-cancel.png</normaloff>:/res/img/16x16/settings-cancel.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupShopSettings">
     <property name="title">
      <string>Настройки магазина</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QLabel" name="labelFtpAddress">
        <property name="text">
         <string>Адрес FTP сервера</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="labelPort">
        <property name="text">
         <string>Порт</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelshopname">
        <property name="text">
         <string>Название магазина</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelPeriodCheck">
        <property name="text">
         <string>Период проверки (клипы)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="ftpAddressField">
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QSpinBox" name="portField">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>21</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QSpinBox" name="periodCheckField">
        <property name="minimum">
         <number>5</number>
        </property>
        <property name="maximum">
         <number>180</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLineEdit" name="shopnameField"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelUsername">
        <property name="text">
         <string>Имя пользователя</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelPassword">
        <property name="text">
         <string>Пароль</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="3">
       <widget class="QLineEdit" name="usernameField"/>
      </item>
      <item row="4" column="1" colspan="3">
       <widget class="QLineEdit" name="passwordField">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../res.qrc"/>
 </resources>
 <connections/>
</ui>
